assets/css/sensei-course-theme/quiz-compat.scss

```diff
@@ -0,0 +1,15 @@
+.sensei-course-theme__quiz {
+	&__main-content {
+		.wp-block-post-title {
+			margin-bottom: 2.5rem;
+
+			@media (max-width: 499px) {
```

Comment:
We should always use `min-width` queries as they are mobile first and a best practice.

Also, LM switches to mobile view at `782px` and not `499px`. It would be best to use [Gutenberg's breakpoints](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_breakpoints.scss) instead of hardcoding them. In this case, we can use `$break-medium`. We are already using these breakpoints elsewhere in the codebase, so you should only need to import and then use.

Reply:
Very good suggestion, wasn't aware of them before. I only knew about declaring variables ourselves like this https://github.com/Automattic/sensei/blob/db7f89cc6171961b419db45a98dea30104bf8eef/assets/css/sensei-course-theme/fixed-sidebar.scss#L1. Thank you so much for the 'TIL'! ❤️ 

Updated it here https://github.com/Automattic/sensei/pull/7093/commits/a2d416b0112f8cf773fd454e3414c5f48c12227f